src/index.ts

```diff
@@ -14,6 +15,31 @@ const PLUGIN_NAME = 'postcss-react-strict-dom'
 
 const builder = createBuilder()
 
+function getDefaultInclude() {
+  let reactStrictDomPackageJsonFilePath: string
+  try {
+    reactStrictDomPackageJsonFilePath = require.resolve(
+      'react-strict-dom/package.json'
+    )
+  } catch (error) {
+    throw new Error(
+      `Could not find the "react-strict-dom" package. Make sure it is installed in your project.`
+    )
+  }
+  return [
+    // Include the React Strict DOM package's source files by default
+    path.join(reactStrictDomPackageJsonFilePath, 'dist/**/*.{js,jsx,ts,tsx}'),
```

Comment:
It might be better to point only to the DOM / default entry

Reply:
I think this actually needs to be `require.resolve('react-strict-dom')`. That will return the path to the web module. This currently returns a broken path